lib/spack/spack/util/compression.py

```diff
@@ -235,7 +274,15 @@ def _7zip(archive_file):
     Args:
         archive_file (str): absolute path of file to be unarchived
     """
+    # 7zip will extract .txz atomically
+    # but not .tar.xz, rename tar.xz
+    # to .txz for atomic extraction
```

Comment:
I understand what you mean here but I don't think "extract" is strictly defined to mean unarchiving the .tar, so can you clarify that in the comment?

Reply:
No problem.